### PR TITLE
Add rubric summary

### PR DESCRIPTION
**Pull request by kieranderson**
Adding rubric summary to the evaluation accordion.

### CODE REVIEW

File: components/d2l-activity-editor/d2l-activity-rubrics/d2l-activity-rubrics-summary.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+		if (!entity) {
+			return html``;
+		}
+
+		const associationsCount = entity.fetchAttachedAssociationsCount();
+		if (associationsCount <= 0) {
+			return html`${this.localize('txtNoRubricAdded')}`;
+		}
+
+		return html`${this.localize('txtRubricsAdded', 'count', associationsCount)}`;
```
Comment (ppaskaris-d2l) [inline components/d2l-activity-editor/d2l-activity-rubrics/d2l-activity-rubrics-summary.js]: this works? I thought it was `this.localize('txtRubricsAdded', {count: associationsCount})`

File: components/d2l-activity-editor/d2l-activity-rubrics/state/association-collection.js [hunk 2]
```diff
@@ -44,6 +44,19 @@ export class AssociationCollection {
 		return Array.from(this.associationsMap.values());
 	}
 
+	fetchAttachedAssociationsCount() {
+		const associations = Array.from(this.associationsMap.values());
+		let attachedAssociationCount = 0;
+		associations.forEach(association => {
```
Comment (ppaskaris-d2l) [inline components/d2l-activity-editor/d2l-activity-rubrics/state/association-collection.js]: Consider `for-of` against `this.associationsMap.values()` directly.

File: components/d2l-activity-editor/d2l-activity-rubrics/d2l-activity-rubrics-summary.js (hunk 1 above)
Comment (pureooze) [reply components/d2l-activity-editor/d2l-activity-rubrics/d2l-activity-rubrics-summary.js]: Yes it does! https://github.com/BrightspaceHypermediaComponents/activities/blob/master/components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-type-editor.js#L127

File: components/d2l-activity-editor/d2l-activity-rubrics/state/association-collection.js (hunk 2 above)
Comment (kieranderson) [reply components/d2l-activity-editor/d2l-activity-rubrics/state/association-collection.js]: 👍 Will update in next PR.

**Pull request merged**